scripts/ci/automation_full_test.py

```diff
@@ -15,28 +15,165 @@
 ch = logging.StreamHandler()
 ch.setLevel(logging.DEBUG)
 logger.addHandler(ch)
-profile = sys.argv[1]
-serial_modules = sys.argv[2].split()
+
+instance_cnt, instance_idx = [int(i) for i in sys.argv[1].split('_')]
```

Comment:
Could you explain where `sys.argv` is passed?

Reply:
sys.argv is passed by .azure-pipelines/templates/automation_test.yml in section `Running full test`